test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -454,12 +454,12 @@ upgrade_downgrade()
             echo "$1 format running on $2 access method $am..."
             cd "$top/$format_dir_branch1"
             flags="-1Rq $(bflag $1)"
-            ./t $flags -c "$top/$format_dir_branch2/${cfg_file_branch1}" -h "$top/$format_dir_branch2/RUNDIR.$am" timer=2
+            ./t $flags -h "$top/$format_dir_branch2/RUNDIR.$am" timer=2
 
             echo "$2 format running on $2 access method $am..."
             cd "$top/$format_dir_branch2"
             flags="-1Rq $(bflag $2)"
-            ./t $flags -c $cfg_file_branch2 -h "RUNDIR.$am" timer=2
+            ./t $flags -h "RUNDIR.$am" timer=2
```

Comment:
Is it correct to say that these `-c` options were always being ignored due to `-R` being in the flags? Also, we no longer need the `cfg_file_branchX` variables a few lines up from here.

Reply:
Yes, we ignore the provided config file in the case of '-R` and use the config inside the home directory. 
Thanks, removed the config variables.